cty/walk_test.go

```diff
@@ -101,11 +101,11 @@ func TestTransform(t *testing.T) {
 		"unknown_map":  UnknownVal(Map(Bool)),
 	})
 
-	gotVal, err := Transform(val, func(path Path, val Value) (Value, error) {
-		if val.Type().IsPrimitiveType() {
+	gotVal, err := Transform(val, func(path Path, v Value) (Value, error) {
+		if v.Type().IsPrimitiveType() {
```

Comment:
This code isn't really affected, but I decided to change it anyway to prevent the ambiguity and similar bugs in the future, in case anyone copies from here.